### PR TITLE
feat: [#176167278] Show message when remove profile if some bonus is active

### PR DESCRIPTION
**Pull request by debiff**
## Short description
This PR show an alert to inform the user that has some bonus active when choice to remove the profile.
If the user hasn't any active bonus the flow it's unchanged.

## List of changes proposed in this pull request
- Added alert if the user that are deleting the profile has at least one active bonus
-Request bonus information with a new saga if the user are deleting his profile and the bonus information aren't loaded yet.

### CODE REVIEW

File: ts/screens/profile/RemoveAccountInfoScreen.tsx [hunk 1]
```diff
@@ -46,5 +46,7 @@ const RemoveAccountInfo: React.FunctionComponent<Props> = props => {
     </BaseScreenComponent>
   );
 };
-
-export default RemoveAccountInfo;
+const mapDispatchToProps = (dispatch: Dispatch) => ({
+  loadBonusAndProceed: () => dispatch(loadBonusBeforeRemoveAccount())
```
Comment (Undermaken) [inline ts/screens/profile/RemoveAccountInfoScreen.tsx:50]: What happen if the API response with a delay? we press the button but nothing happens
To take it easy I suggest to remove the navigation from saga `handleLoadBonusBeforeRemoveAccount` and place it there.
So you at the same time

- ask for bonus data
- navigate to the next screen

If we can't load data the alert won't be shown. But I guess it is acceptable

**Review by Undermaken — CHANGES_REQUESTED** [review]

File: ts/sagas/startup.ts [hunk 2]
```diff
@@ -262,6 +264,11 @@ export function* initializeApplicationSaga(): Generator<Effect, void, any> {
     backendClient.getSupportToken
   );
 
+  // Sart watching for request of remove profile
+  yield takeEvery(
```
[the remark this answers is not in the document]
Comment (debiff) [reply ts/sagas/startup.ts:268]: Agree

File: ts/screens/profile/RemoveAccountDetailsScreen.tsx [hunk 3]
```diff
@@ -68,22 +72,59 @@ const RemoveAccountDetails: React.FunctionComponent<Props> = (props: Props) => {
...
+            }
+          }
+        ]
+      );
+    } else {
+      handleSendMotivaiton(selectedMotivation);
+    }
+  };
+
+  const handleSendMotivaiton = (
```
Comment (Undermaken) [inline ts/screens/profile/RemoveAccountDetailsScreen.tsx:109]:
```suggestion
  const handleSendMotivation = (
```

File: ts/store/actions/profile.ts [hunk 4]
```diff
@@ -48,6 +48,10 @@ export const profileFirstLogin = createStandardAction("PROFILE_FIRST_LOGIN")();
 
 export const clearCache = createStandardAction("CLEAR_CACHE")();
 
+export const loadBonusBeforeRemoveAccount = createStandardAction(
```
Comment (Undermaken) [inline ts/store/actions/profile.ts:51]: Could you leave a comment describing why this action is needed?

**Review by Undermaken — CHANGES_REQUESTED** [review]

**Review by Undermaken — APPROVED** [review]

**Pull request merged**